comp/forwarder/defaultforwarder/domain_forwarder_test.go

```diff
@@ -421,6 +421,8 @@ forwarder_requeue_buffer_size: 1300
 }
 
 func newDomainForwarderForTest(config config.Component, log log.Component, connectionResetInterval time.Duration, ha bool) *domainForwarder {
+	config.SetWithoutSource("forwarder_max_concurrent_requests", 1)
```

Comment:
Would it be valuable to update the tests to not rely on ordering? The answer can be no 🙂

Reply:
I added the TestHighPriorityTransactionTendency test below that does this by testing that on average we will send more high priority first than low priority.